airflow/settings.py

```diff
@@ -188,6 +188,16 @@ def configure_adapters():
         pass
 
 
+def configure_action_logging():
+    """
+    Any additional configuration for airflow.utils.action_loggiers module
+    e.g: Use airflow.utils.action_loggers.add(action_logger) to register callback
```

Comment:
Would be nice to know what `action_logger` should expect to receive as params in this context

Reply:
Also the term callback is a bit misleading here as the interface here is a class that has a submit `@classmethod`